extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -59,12 +59,11 @@ import {getContextMetadata} from '../../../src/iframe-attributes';
 // import '../../amp-ad/0.1/amp-ad-ui';
 // import '../../amp-ad/0.1/amp-ad-xorigin-iframe-handler';
 
-/** @type {string} */
-const METADATA_STRING = '<script type="application/json" amp-ad-metadata>';
-
-/** @type {string} */
-const METADATA_STRING_NO_QUOTES =
-      '<script type=application/json amp-ad-metadata>';
+/** @type {Array<string>} */
+const METADATA_STRINGS = [
+  '<script amp-ad-metadata type=application/json>',
+  '<script type="application/json" amp-ad-metadata>',
+  '<script type=application/json amp-ad-metadata>'];
```

Comment:
I'm wondering whether we just want to look for the first string, i.e., `<script amp-metadata type=...>`.  According to an internal discussion in May, attribute names should always be sorted, and quotes should be omitted. I'm not sure that's become officially specified yet though.